web-frontend/src/pages/TransferCoins/SendCoins/Advanced.tsx

```diff
@@ -52,12 +52,14 @@ function Modal({ ...props }) {
   function handlePresetGas() {
     setCustomFees(false);
     setPresetGasFees(true);
+    handleFees('1000');
```

Comment:
hmm so as soon as we select the radiobutton it overrides the fees ?

Reply:
yeah and sets it to standard fees